_data/milestones.yml

```diff
@@ -195,7 +195,7 @@
   long: James Fergason využívá objev z roku 1888 a sestavuje první moderní LCD displej.
 - time: '1970'
   short: Optické vlákno
-  long: Vědci Kapany a Kao od padesátých let pracují na technologiích vedoucích k modernímu optickému vláknu.
+  long: Vědci N.S.Kapany a Ch.K.Kao od padesátých let pracují na technologiích vedoucích k modernímu optickému vláknu.
```

Comment:
Rovnou vyzkouším i komentář :) Za těmi iniciálami by určitě neměla chybět mezera.